OpenUpSA/wazimap-ng-ui
# handling missing data property

## Description
Checking if `data` property is available in `utils.js`

## Related Issue


## How to test it locally
* set hostname as `sandbox.sanef-local-gov.openup.org.za`
* navigate to a province and confirm front end does not log any error on developer console


## Screenshots


## Changelog

### Added

### Updated
* added an extra check to confirm the `data` property is available

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
